lib/matplotlib/textpath.py

```diff
@@ -104,22 +104,46 @@ def get_text_width_height_descent(self, s, prop, ismath):
 
     def get_text_path(self, prop, s, ismath=False, usetex=False):
         """
-        convert text *s* to path (a tuple of vertices and codes for
+        Convert text *s* to path (a tuple of vertices and codes for
         matplotlib.path.Path).
 
-        *prop*
-          font property
+        Parameters
+        ----------
 
-        *s*
-          text to be converted
+        prop : `matplotlib.font_manager.FontProperties` instance
+            The font properties for the text.
 
-        *usetex*
-          If True, use matplotlib usetex mode.
+        s : string
+            The text to be converted.
+            
+        usetex : bool
+            If True, use matplotlib usetex mode.
```

Comment:
Do we have a link or a description what "matplotlib usetex mode" is?

Reply:
Good question. I now replaced it with 

> Whether to use tex rendering. Defaults to ``False``.

same as further down in `TextPath`.